packages/modules/web_themes/colors/source/src/components/NavigationBar.vue

```diff
@@ -20,6 +20,7 @@
 						id="navStatus"
 						class="nav-link"
 						href="../../settings/#/Status"
+						target="_parent"
```

Comment:
Ich war so frei und hatte diese Parameter letztens entfernt, da sie für Nebeneffekte gesorgt haben. Das waren noch Überbleibsel, als ich zuerst das Theme in einem IFrame laden wollte, was jedoch nicht funktioniert hat.

Kannst Du bitte auf den aktuellen Master rebasen?

Reply:
Hm, da muss ich erstmal rausfinden wie das geht...

Reply:
In diesem Fall sollte es ausreichen, wenn Du die Zeilen entfernst.